cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/service/BoardsConfigService.kt

```diff
@@ -30,11 +36,16 @@ class BoardsConfigService(
         contentRepository.save(widgetId, content)
     }
 
+    fun setDeletionHelper(helper: BoardDeletionHelper) {
+        boardDeletionHelper = helper
+    }
+
     private fun getWidgetById(boardsConfig: JsonObject) =
             boardsConfig.getJsonObject("widgets")
                     ?.getJsonObject("widgetsById") ?: JsonObject()
 
     private fun clearContent(widgetId: String, widget: JsonObject) {
```

Comment:
maybe we should rename this to `resetContentNode`

Reply:
Seconded, will rename.